### ISSUE TITLE
[FEATURE] Posts after date flag

### ISSUE THREAD

**Issue by ch8zer** [aliparlakci/bulk-downloader-for-reddit#680]
- [X ] I am requesting a feature.
- [ ] I am running the latest version of BDfR
- [ ] I have read the [Opening an issue](../../README.md#configuration)
  
## Description

It would be nice if there was a flag such as `--max-age 2d` which indicates to skip posts older than 2 days.

For example, let's say I scrape a subreddit that has 3 posts, and I run it with `--max-age 2d` given that today is "2022-01-15"

| POST | AGE  | Downloaded |
|---|---|---|
| A | 2022-01-01 | no |
| A | 2022-01-12 | yes |

**Comment by OMEGARAZER** [comment]
While "possible" with a re-write of some things, you can use --time [for the --sort options it's available with](https://github.com/aliparlakci/bulk-downloader-for-reddit/discussions/607#discussioncomment-2248960) but your best bet is probably to use --exclude-id-file with [these](https://github.com/aliparlakci/bulk-downloader-for-reddit/tree/development/scripts) so that nothing would be redownloaded.

**Comment by Serene-Arc** [comment]
Yeah the only way to do this would lower the amount of submissions that you can get overall. It'd be like this: Reddit gives us a thousand posts and we weed out whatever is older, say 300 posts. That gives you 700 posts, not 1000 older posts. Reddit doesn't have the option to make age-limited searches so neither does the BDFR.

In other words, were this feature to be implemented then it would be the same as you putting the post date in the filename, and then deleting the files that are older than what you want. Hardly the optimal solution, but it's the best we can do.

Given that this would dramatically cut down on downloads I'm willing to implement it though.

**Comment by TimidLo** [comment]
I'm also curious about this - If one uses the script, passes that file onto `--exclude-id-file`, and then reruns the app, it'll skip over those items. But does the new log file from the re-run contain information from these excluded files, so that when I run the script again, it'll output a monotonically increasing file of all previous downloads? Or do I need to save every log file and concatenate the outputs together to have a record of all the saved posts so I don't redownload them?

Just wondering what the best strategy is to run this script over particular subreddits that I want to scrape, while ignoring all data I have downloaded in the past from multiple runs of this app?

**Comment by Serene-Arc** [comment]
You need to concatenate it, *if the scripts provided*. Technically speaking, the BDFR does output the the skipped submission's ID so you can use a script to find all of those from the log files, but the scripts provided don't search for that.
